next/package.json

```diff
@@ -9,16 +9,37 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "next": "^12.3.4",
+    "@clevercanary/data-explorer-ui": "0.21.0",
+    "@emotion/react": "11.10.4",
+    "@emotion/styled": "11.10.4",
+    "@mui/icons-material": "5.8.0",
+    "@mui/material": "5.8.1",
+    "imagemin-mozjpeg": "^9.0.0",
+    "imagemin-optipng": "^8.0.0",
+    "imagemin-svgo": "^9.0.0",
+    "next": "12.3.1",
+    "next-optimized-images": "^2.6.2",
     "react": "17.0.2",
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "@next/eslint-plugin-next": "^13.1.6",
+    "@typescript-eslint/eslint-plugin": "^5.49.0",
     "@types/node": "20.2.3",
     "@types/react": "17.0.2",
```

Comment:
Hi @BruceRodrigues can you please check that each dependency and dev-dependency release versions match the data-browser. e.g. `@types/react` version should be `"^18.0.15"`. Thank you!